Milind radio water

Milind21 opened the pull request.

Milind21 committed:
Added Radio Feature and Animation for watering

- Added water droplets for animating watering a plant. (Probably separating initial location of plant will be good)
- added radio, (Press YInput to cycle the stations and B input to play initially )

Milind21 committed:
Added Animation for music

- Added animation for music

Milind21 committed:
Updated Src file

- Added music icon

Milind21 committed:
Added stop feature

- Press AInput to stop the radio or simply move away from it (that is raycast doesnt hit it)

blst15ki commented on Assets/Scripts/RadioManager.cs:
```diff
@@ -0,0 +1,52 @@
+                    audioSource.Play();
+                    mainCamera.transform.Find("MusicParticleSystem").GetComponent<ParticleSystem>().Play();
+                }
+                else if (Input.GetButtonDown(YInput)){
```
You can play music without any visual effects by pressing the Y button on the radio.

I'm also not sure what the purpose of adding an animation for this is. If you want music particles, they should play from the radio instead of from the camera. I'd suggest creating a particle system on top of the radio and adjusting it to emit particles upward.

blst15ki commented on Assets/Scripts/RadioManager.cs:
```diff
@@ -0,0 +1,52 @@
+            }
+            else{
+                hotbar.EnableHotbar();
+                audioSource.Stop();
```
Music can only be played as long as the user is looking at the radio. Is this intentional? I think it would be better to keep playing music until the user leaves the scene or turns off the radio.

blst15ki commented on Assets/Scripts/RadioManager.cs:
Music ends after the song is done playing. Assuming you agreed with keeping the music on even if the user is not looking at the radio, there is a loop option in the inspector window for the audio source component to keep playing music until the user stops the song.

blst15ki commented on src.txt:
You need to specifically cite the music you're using and the radio asset

Milind21 committed:
Addressed comments

- Music plays until user changes scene or stops it (AInput)
- The music loops over and doesnt stop
- Animation comes from the radio as long as the music is playing

Milind21 committed:
Updated src.txt

- Added the music sources for each track.

Milind21 committed:
Updated src.txt (Added Radio Asset)

blst15ki committed:
Fix music particle system, update detector and radio manager
